src/app/+item-page/edit-item-page/modify-item-overview/modify-item-overview.component.html

```diff
@@ -7,10 +7,12 @@
   </tr>
   </thead>
   <tbody>
-  <tr *ngFor="let metadatum of metadata" class="metadata-row">
-    <td>{{metadatum.key}}</td>
-    <td>{{metadatum.value}}</td>
-    <td>{{metadatum.language}}</td>
-  </tr>
+  <ng-container *ngFor="let entry of metadata | keyvalue">
+    <tr *ngFor="let value of entry.value" class="metadata-row">
+      <td>{{entry.key}}</td>
+      <td>{{value.value}}</td>
```

Comment:
I dislike having `value.value` I suggest to rename `entry `to `mdItem` and `value` to `mdValue`. Then you will have `mdValue.value`